### DISCUSSION TITLE
Contributions - Code Style

### DISCUSSION THREAD

**Discussion by jchue** [SamTV12345/PodFetch#173] [Q&A]
Hi, is there a code style guide you'd like to follow?

**Comment by SamTV12345** [comment]
There is currently no code style. I use IntelliJs auto configuration.

**Reply by SamTV12345** [reply]
So indentation should be 4 across the files.

**Comment by jchue** [comment]
Sounds good! And semicolons or no?

**Reply by SamTV12345** [reply]
Please no semicolons. For me they don’t have any function and it  just makes the code more cluttered.
